### PR TITLE
Revert "Revert "Introduce Intercom chat button""

### PR DESCRIPTION
**Pull request by akito19**
Reverts sider/sider-docs#166

### CODE REVIEW

File: website/core/plugins/Intercom.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+
+const appId = process.env.NODE_ENV === 'development' ? "671su9bf": "rpwp2i8e";
```
Comment (akito19) [inline website/core/plugins/Intercom.js]: When set `production`, development version chat button is shown in help.sider.review.
I don't know why the thing has happened, so I use `development` instead...

Comment (ybiquitous) [reply website/core/plugins/Intercom.js]: See https://www.netlify.com/docs/build-settings/#build-environment-variables

Comment (akito19) [reply website/core/plugins/Intercom.js]: Hmm...
Netlify's behavior was a cause 🙃

File: website/core/plugins/Intercom.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+
+const appId = process.env.CONTEXT === 'production' ? "rpwp2i8e" : "671su9bf";
```
Comment (akito19) [inline website/core/plugins/Intercom.js]: Try `CONTEXT` environment variable to check whether the document has been hosted in "production". 
https://www.netlify.com/docs/continuous-deployment/?_ga=2.165339647.1123237602.1559186784-507422736.1558931826#environment-variables

If the way is failed, I'll try another way.

**Comment by akito19** [comment]
It seems to be fine 👏 
Screen Shot 2019-05-31 at 14 50 53

**Comment by ybiquitous** [comment]
Nice! It seems Docusaurus processes environment variables well… (but I'm not sure) 🤔

**Pull request merged**